src/commands/start.js

```diff
@@ -58,7 +58,7 @@ function execute(collectorName, refocusUrl, accessToken, rcProxy) {
      * logging once we have heartbeat
      */
     repeater.create({
-      name: 'Heartbeat',
+      name: 'heartbeat',
```

Comment:
move this hard-coded constant to the constants file?

Reply:
@iamigo Can you merge this? I will do this in #112